dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -647,19 +647,45 @@ function onSkinTemplateNavigationUniversal( SkinTemplate $skin, array &$links )
     // T309379
     $wgCirrusSearchEnableArchive = false;
     $wgCirrusSearchPrivateClusters = [ 'non-existing-cluster' ];
-    
-    $wgCirrusSearchClusters = [
-        'default' => [
-            [
-                'transport' => [
-                    'type' => \CirrusSearch\Elastica\ES6CompatTransportWrapper::class,
-                    'wrapped_transport' => 'Http'
+
+    function getElasticClusterConfig( string $prefix ) {
+        $config = [
+            'host' => getenv( $prefix . 'HOST' ),
+            'port' => getenv( $prefix . 'PORT' )
+        ];
+
+        if ( getEnv( $prefix . 'ES6' ) ) {
```

Comment:
Very nitpicky, but this does not work as described in the README (i.e. it does not check if it's `"true"`).